src/main/resources/lib/github-checks/status/properties.jelly

```diff
@@ -9,17 +9,19 @@
     <f:textbox default="Jenkins"/>
   </f:entry>
 
+
   <f:entry title="${%Publish unstable builds as neutral status checks}" field="unstableBuildNeutral">
     <f:checkbox/>
   </f:entry>
 
   <f:entry title="${%Suppress log output in checks}" field="suppressLogs">
     <f:checkbox/>
   </f:entry>
-
-  <f:entry title="${%Suppress progress updates in job check}" field="skipProgressUpdates"
-           description="${%Queued, checkout and completed will still be sent}">
-    <f:checkbox/>
-  </f:entry>
-
+  <j:var set="readOnlyMode" value="${descriptor.enforceSkipProgressUpdates}" />
+  <j:if test="${!readP}">
+    <f:entry title="${%Suppress progress updates in job check}" field="skipProgressUpdates"
+             description="${%Queued, checkout and completed will still be sent}">
+      <f:checkbox default="${descriptor.defaultSkipProgressUpdates}"/>
+    </f:entry>
+  </j:if>
```

Comment:
We would need some advice on Jelly conditional rendering here.. we try to hide the option on the jobs, if the global configuration says that the option is enforced.
It seems like ${!readOnlyMode} does not resolve correctly here (we tried it with and without the descriptor). If you have any idea, let us know. Thanks!